README.md

```diff
@@ -32,6 +32,11 @@ If you need to install Go, follow [the official instructions](https://golang.org
 
 Find up-to-date examples and documentation on [godoc.org](https://godoc.org/github.com/Azure/azure-service-bus-go#pkg-examples).
 
+### Running tests
```

Comment:
The Makefile (`make test`) should cause the infrastructure to be created and place a .env file in the local directory upon the first execution. This should be the way folks get started with tests and infrastructure.

If these notes are for Windows devs, I'd suggest adding a dockerfile w/ a suitable go environment via a command like `make dev` to get a shell with the source code mounted. Then from there, Windows devs can use the same `make test` from within the container.

Telling folks to try to replicate the scripts in the Makefile is pretty painful.

Reply:
Oh I didn't know about `make test`, I'll rewrite this section accordingly.

It would be nice at some point to have a PS script to run the tests natively on Windows.

Reply:
I started down that path in #92, but never really followed through on it. I don't think it would too tough to get the 90% case down. At the same time, I was conflicted about adding PS scripting rather than going down the path of `make dev` and container based build env for Windows and possibly build environments. At this point, I feel like the container based build would be better than keeping up two separate build scripts to run natively on both Windows and Unix'ish OSes. Users have VS Code extensions to do remote dev in Unix'ish environments from Windows, remote debugging, etc...

Reply:
I agree with the maintaining of two scripts.  However I've found there is value in being able to run tests end-to-end in both environments as it catches inadvertent dependencies on OS-specific behavior (directory separator being a common thing).  Perhaps that's less relevant in this module though.

Reply:
Apparently the makefile has other dependencies (gocyclo).  Plus I think we still have to set environment variables.  For now I'm going to leave this as-is so I can get this release out the door then update these instructions in a separate PR.